helper.py

```diff
@@ -0,0 +1,30 @@
+# Copies files and folders to relevant containers
+import os
+
+
+def cp_helper(name, test_name, container):
+    if test_name == "storage":
```

Comment:
Uhm... Not very happy about the switch according to the test_name here.
Can we move this logic outside and always pass to the function the full container name and additional params (such as name of the user) on the need?

Reply:
Done!